cl_manycore/regression/spmd/spmd_tests.h

```diff
@@ -18,6 +18,7 @@
 #include <bsg_manycore_mem.h>
 #include <bsg_manycore_loader.h>
 #include <bsg_manycore_errno.h>
+
```

Comment:
Whitespace change?

Reply:
Well it wasn't hurting anybody but I fixed it anyway :D